### PR TITLE
prepare v8 release

### PR DESCRIPTION
**Pull request by okwme**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,13 +35,16 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 * (gaia) bump [Liquidity](https://github.com/Gravity-Devs/liquidity) module to [v2.0.1](https://github.com/Gravity-Devs/liquidity/releases/tag/v2.0.1).
 * (gaia) bump [ibc-go](https://github.com/cosmos/ibc-go) module to [v5.0.1](https://github.com/cosmos/ibc-go/releases/tag/v5.0.1). See [CHANGELOG.md](https://github.com/cosmos/ibc-go/blob/v5.0.1/CHANGELOG.md) for details.
 * (gaia) bump [packet-forwarding-middleware](https://github.com/strangelove-ventures/packet-forward-middleware) to depend on ibc-go v5.
-* (gaia) Update go to 1.18.
-* (feat) Add [Groups](https://docs.cosmos.network/main/modules/grthaoup/#group-module) module.
+* (gaia) update go to 1.18.
+* (feat) add [Groups](https://docs.cosmos.network/main/modules/grthaoup/#group-module) module.
 * (feat) add [icamauch](https://github.com/cosmos/gaia/tree/main/x/icamauth) (Interchain Account Message Authorization) module.
 * (feat) add [globalfee](https://github.com/cosmos/gaia/tree/main/x/globalfee) module.
 * (feat) replace `MempoolFeeDecorator` with [FeeDecorator](https://github.com/cosmos/gaia/blob/main/x/globalfee/ante/fee.go) to check fees.
```
Comment (yaruwangway) [inline CHANGELOG.md:49]: there is no changelog for v7.0.3  and v7.1.0 ?

Comment (yaruwangway) [reply CHANGELOG.md:49]: Hi I added, please check

Comment (okwme) [reply CHANGELOG.md:49]: thank you @yaruwangway !

File: CHANGELOG.md [hunk 2]
```diff
@@ -445,7 +456,10 @@ See the [Tendermint v0.34.7 SDK changelog](https://github.com/tendermint/tenderm
 
 <!-- Release links -->
 
-[Unreleased]: https://github.com/cosmos/gaia/compare/v7.0.3...HEAD
+[Unreleased]: https://github.com/cosmos/gaia/compare/v8.0.0...HEAD
+[v8.0.0]: https://github.com/cosmos/gaia/releases/tag/v8.0.0
```
Comment (mmulji-ic) [inline CHANGELOG.md:462]: Should we have the RC link, until the real release gets made?

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,13 +35,15 @@ Ref: https://keepachangelog.com/en/1.0.0/
 # Changelog
 
 ## [Unreleased]
+
+
+## [v8.0.0] - 2022-10-31
 * (gaia) bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to [v0.46.3](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.3). See [CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/v0.46.3/CHANGELOG.md) for details.
```
Comment (mmulji-ic) [inline CHANGELOG.md]: SDK now 0.46.4

File: CHANGELOG.md (hunk 2 above)
Comment (okwme) [reply CHANGELOG.md:462]: was never done before, i think it should be expecting the next actual release. The point is to create compare links so people can see the difference? https://github.com/olivierlacan/keep-a-changelog/issues/71

File: CHANGELOG.md (hunk 3 above)
Comment (okwme) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 2 above)
Comment (mmulji-ic) [reply CHANGELOG.md:462]: Ok lets keep as is.

**Review by mmulji-ic — APPROVED** [review]

**Pull request merged**